### DISCUSSION TITLE
Support 120 Hz refresh rate when scrolling

### DISCUSSION THREAD

**Discussion by LostInCompilation** [brackets-cont/brackets#214] [Feature Request]
When scrolling on the M1 Pro MacBook Pro 16" with ProRes Display, Brackets is clearly not using 120Hz refresh rate and looks quite choppy. Please support high refresh rates for scrolling.

**Comment by abose** [comment]
Thanks for reporting the issue. This will be addressed in brackets 3.0.

**Comment by charlypa** [comment]
@LostInCompilation 
We have modernized brackets. We released it yesterday. The new editor is Phoenix Code, Now you can download native m1 and intel dmg instead of relying on old Brackets x86 binaries
visit https://phcode.io/
